src/js/pages/services/ServicesTab.js

```diff
@@ -210,10 +226,11 @@ var ServicesTab = React.createClass({
             <ServiceSearchFilter
               handleFilterChange={this.handleFilterChange} />
             <button className="button button-stroke button-inverse"
-                    onClick={this.handleOpenGroupFormModal}>
+              onClick={() => this.handleOpenModal(SERVICE_GROUP_FORM_MODAL)}>
               Create Group
             </button>
-            <button className="button button-success">
+            <button className="button button-success"
+              onClick={() => this.handleOpenModal(SERVICE_FORM_MODAL)}>
```

Comment:
Well it is in other places e.g. [HealthTab.js:118](https://github.com/dcos/dcos-ui/blob/master/src/js/components/HealthTab.js#L118)